play-media-native/screens/EditMedia.tsx

```diff
@@ -122,12 +107,12 @@ export const EditMediaScreen = ({ navigation, route }) => {
         <Button
           mode="contained"
           onPress={onAdd}
-          // labelStyle={styles.buttonLabel}
-          // style={styles.button}
+          labelStyle={styles.buttonLabel}
+          style={styles.button}
         >
           {isEdit ? "Edit Media" : "Add Media"}
         </Button>
       </BottomActions>
-    </SafeAreaView>
+    </KeyboardAwareScrollView>
```

Comment:
Just a minor thing, description text input is not visible when focused 
![image](https://user-images.githubusercontent.com/96472119/215829010-2442ea13-df96-467c-858e-1952f25fd67c.png)

Maybe hide the BottomActions when keyboard is visible?

Reply:
Isn't it possible to scroll down to it?

Reply:
Nope, not possible in the emulator.